src/main/java/thePackmaster/cards/startuppack/Cross.java

```diff
@@ -0,0 +1,63 @@
+package thePackmaster.cards.startuppack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.startuppack.CrossPower;
+import thePackmaster.util.Wiz;
+
+import java.util.Objects;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Cross extends AbstractStartUpCard {
+    public final static String ID = makeID("Cross");
+    public Cross() {
+        super(ID, 1, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        this.baseDamage = this.damage = 7;
+        this.baseSecondDamage = this.secondDamage = 0;
```

Comment:
```suggestion
        this.baseSecondDamage = this.secondDamage = 0;
        cardsToPreview= new Cross();
```
This way players can see what a Cross is